src/Mappers/InventoryMapper.php

```diff
@@ -71,7 +72,7 @@ public function createInventoryDTOsFromVariation($variationData)
     /** @var LoggerContract $loggerContract */
     $loggerContract = pluginApp(LoggerContract::class);
 
-    /** @var RequestDTO[] $inventoryDTOs */
+    /** @var array $inventoryDTOs */
```

Comment:
Is RequestDTO[] a sniff currently? Not an issue - just curious

Reply:
Also would consider calling this variable `$indexedRequestDTOs` or something - the `array_value` call in the return was a little confusing for me cause this wasn't clear

Reply:
AFAIK `RequestDTO[]` is not a vaild type hint for associative arrays. I could be wrong.
Yeah I agree about the name. Something like `requestDtosBySuID` is more descriptive.

Reply:
found out that I can do `@var array<string,RequestDTO> $requestDtosBySuID`